.gitignore

```diff
@@ -0,0 +1,13 @@
+.DS_Store
```

Comment:
Following #6 should we aim to only use gitignores that come from github/gitignore ?